05week/checkers.js

```diff
@@ -61,9 +115,56 @@ class Game {
   }
   start() {
     this.board.createGrid();
+    this.board.createCheckers();
+  }
+  moveChecker(source, destination){
```

Comment:
This function has a lot of logic.  Think of ways to break this down to smaller functions to make it more readable.  Currently, there is a bug that is preventing the ability to process a jump and there are limitation in the direction in which you can move a piece.